requirements-core.txt

```diff
@@ -6,7 +6,7 @@ lxml==5.2.0
 matplotlib>=3.3.1
 networkx>=2.6
 nibabel>=3.2.1
-numpy==1.26.4
+numpy<2,>=1.23.4
```

Comment:
Before https://github.com/openvinotoolkit/datumaro/commit/017b524632003c43f9899ad4964f5d3e96805aec, numpy version was `>=1.17.3`. Is there any reason to increase this to `1.23.4`?